Competition2017/src/competition/subsystems/climbing/commands/PneumaticBrakeCommand.java

```diff
@@ -0,0 +1,39 @@
+package competition.subsystems.climbing.commands;
+
+import org.apache.log4j.Logger;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.climbing.ClimbingSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class PneumaticBrakeCommand extends BaseCommand {
```

Comment:
instead of this command, let's set the brake based on if power is being set to the climber and if it's stopped set the break on.